### DISCUSSION TITLE
Issue with installing the subfinder

### DISCUSSION THREAD

**Discussion by Dadichi008** [projectdiscovery/subfinder#1441] [General]
![subdomain_error](https://github.com/user-attachments/assets/89d2608b-135d-490e-98f7-b9b50e68cadf)
Facing this issue ever since I've upgraded subfinder. Removed from the root and from all the folders by search option. But I am still facing issue while installing the subfinder. Kindly help me how to overcome this.
Many thanks.

**Comment by mielverkerken** [comment]
You need to add Go bin path to the system paths.
```
echo export PATH=$PATH:$HOME/go/bin >> $home/.bashrc
source $home/.bashrc
```
See [installation-notes](https://docs.projectdiscovery.io/tools/subfinder/install#installation-notes).
